Src/pangen2.c

```diff
@@ -1947,7 +1947,8 @@ generic_case:	fprintf(fd_tt, "\ttrans[%d][%d]\t= ", Pid_nr, e->seqno);
 		goto non_generic;
 #endif
 	case 'c':
-		if (e->n->lft->ntyp == CONST
+		if (e->n->lft->ntyp == CONST 
+		&& e->n->lft->constValKind == VALUE_INT		// TODO PG - verify what to do if float?
```

Comment:
It would be easier to answer this question, if the purpose of this function and this particular case was known. Should we discuss this?

Reply:
as I've already wrote on Teams - I've just marked for myself points that should be further verified if the value can be only int or maybe float should also be expected. As I will go with queues implementation, the semantics of these points will hopefully be understood.